### PR TITLE
Add Aromantic and AroAce flag

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: main.c [hunk 1]
```diff
@@ -323,6 +325,55 @@ const pattern_t unlabeled = {
...
+};
+
+const pattern_t aroace = {
+    .name = "aroace",
+    .ansii_pattern = {
+      208, 208,
+      220, 220,
+      255, 255,
+      75, 75,
+      62, 62
```
Comment (Elsa002) [inline main.c]: They seem to be the wrong values?

File: main.c [hunk 2]
```diff
@@ -323,6 +325,55 @@ const pattern_t unlabeled = {
...
+      }
+    },
+    .color_pattern = {
+      .stripes_count = 5,
+      .stripes_colors = {
+          0x3da542, // Green
+          0xa8d379, // Light green
+          0xffffff, // White
+          0xa9a9a9, // Grey
+          0x000000  // Black
```
Comment (Elsa002) [inline main.c]: Make the comments the same as in the other patterns, with `/* #value - Color */`

File: main.c [hunk 3]
```diff
@@ -323,6 +325,55 @@ const pattern_t unlabeled = {
...
+      62, 62
+    },
+    .color_pattern = {
+      .stripes_count = 5,
+      .stripes_colors = {
+        0xe28d00,
+        0xeccd00,
+        0xffffff,
+        0x62afdd,
+        0x203756
```
Comment (Elsa002) [inline main.c]: Make the comments the same as in the other patterns, with `/* #value - Color */`

File: main.c [hunk 4]
```diff
@@ -323,6 +325,55 @@ const pattern_t unlabeled = {
...
+      },
+      .factor = 1.0f
+    },
+    .get_color = get_color_stripes
+};
```
Comment (Elsa002) [inline main.c:378]: Fix indentations to be 4 spaces and not 2

**Review by Elsa002 — CHANGES_REQUESTED** [review]
Don't forget to add the new flags to the `README.md`

**Comment by ghost** [comment]
I changed all the things you commented on @Elsa002.

**Pull request merged**